src/app/templates/projectPopupTemplate.html

```diff
@@ -0,0 +1,5 @@
+<div class="project-details">
+  <span id="projectStatus-{graphic.attributes.Status}" class="projectStatus pull-right">{graphic.attributes.Status}</span>
```

Comment:
Should be 4 spaces. Not a big deal but thought that you would want to know that something's up with your formatter.